chromadb/db/mixins/embeddings_queue.py

```diff
@@ -243,6 +291,28 @@ def unsubscribe(self, subscription_id: UUID) -> None:
                         del self._subscriptions[topic_name]
                     return
 
+    @trace_method("SqlEmbeddingsQueue.ack", OpenTelemetryGranularity.ALL)
+    @override
+    def ack(self, subscription_id: UUID, up_to_seq_id: SeqId) -> None:
```

Comment:
Here's a wild thought: What if you make notify_all and notify_one return the max sequence ID and thus make this transaction part of the overall submit_embeddings transaction?  Wouldn't that make for a stronger consistency?